packages/network-controller/tests/provider-api-tests/block-hash-in-response.ts

```diff
@@ -364,27 +364,6 @@ export function testsForRpcMethodsThatCheckForBlockHashInResponse(
           await expect(promiseForResult).rejects.toThrow(errorMessage);
         });
       });
-
-      // TODO: Add tests for failover behavior when the RPC endpoint returns a 405 or 429 response without opening a circuit breaker
```

Comment:
We no longer expect the circuit to break for 4XX errors, so failover will never be triggered. These tests are no longer relevant as a result.